pysmt/solvers/msat.py

```diff
@@ -131,20 +131,29 @@ def __contains__(self, x):
         """Returns whether the model contains a value for 'x'."""
         return x in (v for v, _ in self)
 
+
+class MSatOptions(SolverOptions):
+    """Options from MathSAT Solver. """
+    VALID_OPTIONS = SolverOptions.VALID_OPTIONS +\
+                    [("random_seed", None),
```

Comment:
randomseed is part of the SMT-LIB standard, we should move this option to SolverOptions (and incidentally, consider which other options from the standard we can support at this time).